### PR TITLE
Fix expanding arbiter volume

### PR DESCRIPTION
**Pull request by aravindavk**
Arbiter count was not considered when a Volume is expanded.

Fixes: #1161
Signed-off-by: Aravinda VK <avishwan@redhat.com>

### CODE REVIEW

**Comment by aravindavk** [comment]
> @aravindavk Are we looking to add e2e tests to verify this ? Also, I noticed the ''Use:'' section in case of volume expand CLI doesn't contain a few flags usage like --replica, which can be a bit confusing for users, can we modify it in the same PR ?

I will add e2e tests and update the CLI help

**Comment by aravindavk** [comment]
Added e2e test and changed help message. @vpandey-RH @Madhu-1 please review

File: glusterd2/commands/volumes/volume-expand-txn.go [hunk 1]
```diff
@@ -207,13 +207,29 @@ func updateVolinfoOnExpand(c transaction.TxnCtx) error {
 	} else {
 		// Create new Sub volumes with given bricks
 		subvolIdx := len(volinfo.Subvols)
-		for i := 0; i < len(newBricks)/newReplicaCount; i++ {
-			idx := i * newReplicaCount
+		bricksCount := newReplicaCount + volinfo.Subvols[0].ArbiterCount
+		for i := 0; i < len(newBricks)/bricksCount; i++ {
```
Comment (Madhu-1) [inline glusterd2/commands/volumes/volume-expand-txn.go]: can we have `len(newBricks)` calculation outside to avoid computing of this value each time.

Comment (aravindavk) [reply glusterd2/commands/volumes/volume-expand-txn.go]: done

**Pull request merged**